decidim-proposals/spec/commands/decidim/comments/create_comment_with_proposal_links_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Comments
```

Comment:
This shouldn't be necessary if the original command broadcasted an event

Reply:
:thinking:  I think it is required as it tests other things than notifications...

Reply:
But why proposals needs to test a command from comments?

Reply:
@tramuntanal this is still pending, proposals should not test comment creation.

Reply:
Sorry I didn't saw your comment :/

I'm testing a command from comments here because this command can be configured with Proposal's ContentProcessors (that live in proposals module). What I'm testing here is the fact that, when configured as explained, the command behaves as expected: persisting the modified content and broadcasting required notifications.

Reply:
@decidim/lot-core @oriolgual Haves you seen my last comment? Do you still think this spec should be removed?

Reply:
Yes @tramuntanal, see my last comment here, which is relate to the discussion https://github.com/decidim/decidim/pull/2863#issuecomment-371430024